build-logic/convention/build.gradle.kts

```diff
@@ -26,6 +26,9 @@ java {
     sourceCompatibility = JavaVersion.VERSION_17
     targetCompatibility = JavaVersion.VERSION_17
 }
+kotlin {
```

Comment:
Is this necessary or can be deleted?

Reply:
When I wanted to push code , I encountered an error, which was solved by putting this code. I think you can delete it


FAILURE: Build failed with an exception. * What went wrong: Execution failed for task ':build-logic:convention:compileKotlin'. > 'compileJava' task (current target is 17) and 'compileKotlin' task (current target is 18) jvm target compatibility should be set to the same Java version. Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain * Try: > Run with --stacktrace option to get the stack trace. > Run with --info or --debug option to get more log output. > Run with --scan to get full insights. * Get more help at https://help.gradle.org BUILD FAILED in 2s FAILURE: Build failed with an exception. * What went wrong: Execution failed for task ':build-logic:convention:compileKotlin'. > 'compileJava' task (current target is 17) and 'compileKotlin' task (current target is 18) jvm target compatibility should be set to the same Java version. Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain * Try: > Run with --stacktrace option to get the stack trace. > Run with --info or --debug option to get more log output. > Run with --scan to get full insights. * Get more help at https://help.gradle.org BUILD FAILED in 2s failed to push some refs to 'https://github.com/hossein69ameri/ComposeNews.git'